examples/postgres/pooled-with-rustls/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "pooled-with-rustls"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+diesel = { version = "2.0.3", default-features = false, features = ["postgres"] }
+diesel-async = { version = "0.2.0", features = ["bb8", "postgres"] }
```

So that it picks up the diesel-async version from this repository

```suggestion
diesel-async = { path= "../../",  version = "0.2.0", features = ["bb8", "postgres"] }
```